### PR TITLE
Add data workspace datasets

### PR DESCRIPTION
**Pull request by niross**
### Description of change

Adds 3 new data workspace datasets 

- Users
- Event log entries
- Application/tool session

### Checklist

* [ ] Have tests been added to cover any changes?
* [x] Has the [CHANGELOG](https://github.com/uktrade/data-flow/blob/master/CHANGELOG.md) been updated?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

**Comment by niross** [comment]
Apologies for drafting/undrafting - was using the wrong auth credentials so had to rethink things a bit

File: dataflow/dags/data_workspace_pipelines.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            ('owner_id', sa.Column('owner_id', sa.Numeric)),
+            ('proxy_url', sa.Column('proxy_url', sa.Text)),
+            ('public_host', sa.Column('public_host', sa.Text)),
+            ('spawner', sa.Column('spawner', sa.Text)),
+            (
+                'spawner_application_instance_id',
+                sa.Column('spawner_application_instance_id', sa.Text),
+            ),
+            (
+                'spawner_application_template_options',
```
Comment (allait) [inline dataflow/dags/data_workspace_pipelines.py:87]: Is this the JSON field of all the flags for the given app template? I think application template name would be more useful here instead

File: dataflow/dags/data_workspace_pipelines.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    source_url = f'{config.DATA_WORKSPACE_BASE_URL}/api/v1/eventlog/events'
+    table_config = TableConfig(
+        table_name='dataworkspace__event_log',
+        field_mapping=[
+            ('event_type', sa.Column('event_type', sa.Text)),
+            ('id', sa.Column('id', sa.Numeric, primary_key=True)),
+            (('related_object', 'id'), sa.Column('related_object_id', sa.Text)),
+            (('related_object', 'name'), sa.Column('related_object_name', sa.Text)),
+            (('related_object', 'type'), sa.Column('related_object_type', sa.Text)),
```
[the remark this answers is not in the document]
Comment (niross) [reply dataflow/dags/data_workspace_pipelines.py:41]: Type comes from the `DataSet.type` field so would be one of `reference`, `master`, `datacut` or `visualisation` - yeah i'm not sure how much that really adds though...

File: dataflow/dags/data_workspace_pipelines.py (hunk 1 above)
Comment (niross) [reply dataflow/dags/data_workspace_pipelines.py]: Agreed I've swapped it out for `application_template_name` :+1:

**Pull request merged**